test/system/data_entry_test.rb

```diff
@@ -93,6 +98,9 @@ class DataEntryTest < ApplicationSystemTestCase
         assert has_field? 'Patient contribution', with: '150'
         assert has_field? 'National Abortion Federation pledge', with: '50'
         assert has_field? 'DCAF pledge', with: '100'
+        assert has_checked_field? 'Referred to clinic'
+        assert has_checked_field? 'Ultrasound completed?'
+        assert has_unchecked_field? 'Resolved without assistance from DCAF'
```

Comment:
in light of #1841 should this have a thing up near L47 like `check 'patient_resolved_without_fund'` and an associated `assert has_checked_field?` here?

Reply:
Done!